pio/ir_nec/nec_receive_library/nec_receive.pio

```diff
@@ -0,0 +1,94 @@
+;
+; Copyright (c) 2021 mjcross
+;
+; SPDX-License-Identifier: BSD-3-Clause
+;
+
+
+.program nec_receive
+
+; Decode IR frames in NEC format and push 32-bit words to the input FIFO.
+;
+; The input pin should be connected to an IR detector with an 'active low' output.
+;
+; This program expects there to be 10 state machine clock ticks per 'normal' 562.5us burst period,
+; in order to permit timely detection of start of a burst.
```

Comment:
Does that imply the Pico has to be running at a specific frequency, and this won't work if the example program is configured to run with a slower clock-rate? If so, perhaps that's worth mentioning in the README?

Reply:
Yes, I calculate it from the system clock - do you think it's worth me mentioning that in the README?

Reply:
fixed with https://github.com/raspberrypi/pico-examples/pull/129/commits/0162c73b0164ef07f71a68f326f4140781ae25d0